### DISCUSSION TITLE
Continual, repeated requirement to verify email address for users in domain

### DISCUSSION THREAD

**Discussion by johnjames90** [community/community#22334] [New to GitHub]
<p>Users with my specific email domain are finding themselves repeatedly required to verify their email address. This happens every two days or so, even though they are successfully verifying their address through the usual method each time. As expected, they cannot review, push, pull etc. when the yellow banner appears on GitHub, requesting verification.</p>
<p>As a side note, other users within our domain also are presented with this banner, but can still perform actions as usual.</p>
<p>My main query: is there a way to prevent this repeated request for verification? It is a quick task to verify each time admittedly but it is frustrating as emails are regularly received from GitHub with no issue.</p>

**Comment by mpboom** [comment, marked as answer]
<p>Please contact GitHub support if you are experiencing issues with your, or your employees’, GitHub account(s).</p>

**Comment by johnjames90** [comment]
<p>Will do, just thought to check in the community first. Many thanks.</p>
